<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterDialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/background/newBackgrounds.jpg</normaloff>:/background/newBackgrounds.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;
}


#RegisterDialog{
	
	border-image: url(C:/CodingWorkSpace/Icons and Images/Back.jpg);
}

QPushButton{
border-radius: 10px;
border: 2px solid white;
padding: 0 10px;
background-color: rgb(29, 217, 227);

}


QPushButton::hover{
   background-color: rgb(255, 255, 255);
}


QPushButton::hover::pressed{
background-color:rgb(151, 255, 221);
}


QLineEdit
{
background: transparent;
	border-color: rgb(77, 73, 73);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>   REGISTERATION</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="userIDabel">
       <property name="text">
        <string>UserID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="userLineEdit">
       <property name="placeholderText">
        <string>UserID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="placeholderText">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="fNameLabel">
       <property name="text">
        <string>FirstName:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="fNameLineEdit">
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lNameLabel">
       <property name="text">
        <string>LastName:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lNameLineEdit">
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RegDialog.qrc"/>
 </resources>
 <connections/>
</ui>
